TangoRosStreamer/app/src/main/java/eu/intermodalics/tango_ros_streamer/RunningActivity.java

```diff
@@ -305,6 +305,8 @@ public void run() {
                 }
             });
             displayToastMessage(R.string.save_map_success);
+            // Restart Tango to be able to load the new map.
+            mTangoServiceClientNode.callTangoConnectService(TangoConnectRequest.RECONNECT);
```

Comment:
A toast message is displayed when saving the map ended successfully, and another one when reconnecting to Tango is finished.
Maybe we can add the information that we are reconnecting Tango in the first message, e.g.: "Map saved successfully. Tango is restarting now."?